content/java/concepts/calendar/terms/getAvailableCalendarTypes/getAvailableCalendarTypes.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.getAvailableCalendarTypes()'
+Description: 'Returns an unmodifiable Set containing all calendar types supported by Calendar in the runtime environment'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `getAvailableCalendarTypes()` method belongs to the `Calendar` class, which is included in the java.util package. This method returns an unmodified Set that contains all calendar types supported by `Calendar` in the Java runtime environment.
+
+
+## Syntax
+```pseudo
+getAvailableCalendarTypes ()
+```
+> **Note:** The calendar types don't include aliases, such as "gregorian" for "gregory"
+
```

Comment:
```suggestion
## Syntax

```pseudo
getAvailableCalendarTypes()
```

> **Note:** The calendar types don't include aliases, such as "gregorian" for "gregory"

```

Reply:
with line  24 do you want me to remove the note portion?

Reply:
The note can stay